### ISSUE TITLE
LspInstall docker failed

### ISSUE THREAD

**Issue by yriveiro** [anott03/nvim-lspinstall#4]
Running `:LspInstall docker` fails in my machine because is requiring sudo permissions.

OS: Ubuntu 20.10

<img width="973" alt="Screenshot 2021-01-30 at 13 11 29" src="https://user-images.githubusercontent.com/934763/106357265-f8c6f080-62fc-11eb-8907-e81a3fb09cf2.png">

I would assume that all nodejs related language servers will fail as well.

**Comment by anott03** [comment]
Yes. If one nodejs language server requires sudo, they all will. I am working on switching to the method that the lspconfig extension used before deprecating lspinstall, which will install packages to a local directory (most likely `~/.local/share/nvim/lspinstall/node_modules` or something similar). In the meantime, you could copy the command that `LspInstall` is attempting to run, and replace `bash` at the end with `sudo bash`. If you were to do this, the command for Docker would turn into
```bash
curl https://raw.githubusercontent.com/anott03/nvim-lspinstall-scripts/master/Linux/docker.sh | sudo bash
```

**Comment by anott03** [comment]
@yriveiro I believe as of #8 this issue should be resolved.
